src/app/index.js

```diff
@@ -37,6 +37,12 @@ angular.module('app', [
         controller: 'EventController',
         controllerAs: 'vm'
       })
+      .state('university', {
+        url: '/university',
+        template: require('./components/university/universityView.html'),
+        controller: 'UniversityController',
+        controllerAs: 'vm'
+      })
```

Comment:
Let's start giving all of these custom, specific `controllerAs` names. For example, `univCtrl` or `eventCtrl`.